packages/mgt/src/mock/mgt-mock-provider.ts

```diff
@@ -5,18 +5,18 @@
  * -------------------------------------------------------------------------------------------
  */
 
-import { customElement, LitElement, property } from 'lit-element';
+import { property } from 'lit-element';
 import { MgtBaseProvider } from '../components/providers/baseProvider';
 import { Providers } from '../Providers';
 import { MockProvider } from './MockProvider';
+
 /**
  * Sets global provider to a mock Provider
  *
  * @export
  * @class MgtMockProvider
  * @extends {LitElement}
  */
-@customElement('mgt-mock-provider')
```

Comment:
This removes the mock-provider registration without adding an index.ts file. There is no way to use it unless the dev registers it manually.

IMO, mgt-mock-provider should remain as is as it's only used for development

Reply:
Oops, looks like I missed that one. But I agree with you, it doesn't need an extra scoped definition.